src/components/helptexts/HelpTexts.stories.js

```diff
@@ -0,0 +1,55 @@
+import { StoryTitle } from "@/storybook";
+import { HelpText } from "@/components/helptexts/HelpText";
+import { HelpTextMenu } from "@/components/helptexts/menu/HelpTextMenu";
+
+import aHelptText from "./aHelpText.json";
+import allHelp from "./allHelp.json";
+
+export default {
+  title: "helptext/Helptext",
```

Comment:
måske kunne titlen være help/HelpText, for så kan vi andre også smide vores ind i help/blah og få det grupperet samlet